app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,33 @@
+class FoodsController < ApplicationController
+  def index
+    @user = current_user
+    @foods = Food.where(user: current_user) || []
+  end
```

Comment:
- You have done well so far, but according to the project requirements, a logged in user should be able to see all available food irrespective of the owner of the food. Please fix this 🙏. And remember only the owner of a food should be able to delete it.